tests/acceptance/steps/watermark_get.py

```diff
@@ -16,8 +16,7 @@
 
 from behave import then
 from behave import when
-
-from .util import call_watermark_get
+from util import call_watermark_get
```

Comment:
I got a weird "__name__ not in globals python" error when I kept it as a relative import, so I've updated all these imports to use the absolute "steps.util" path